### PR TITLE
Update redirected URLs from `https` -> `http` (part 18)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Extending #14055

## Summary

Updated redirected URLs. Redirecting from `https` to `http`.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/games/techniques/2d_collision_detection/index.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Implementing SAT is out of scope for this page so see the recommended tutorials
 2. [Collision detection and response](https://www.metanetsoftware.com/technique/tutorialA.html)
 3. [Collision detection Using the Separating Axis Theorem](https://gamedevelopment.tutsplus.com/tutorials/collision-detection-using-the-separating-axis-theorem--gamedev-169)
 4. [SAT (Separating Axis Theorem)](https://dyn4j.org/2010/01/sat/)
-5. [Separating Axis Theorem](https://programmerart.weebly.com/separating-axis-theorem.html)
+5. [Separating Axis Theorem](http://programmerart.weebly.com/separating-axis-theorem.html)
```
Comment (teoli2003) [inline files/en-us/games/techniques/2d_collision_detection/index.md:136]: HTTP links are a hazard. Let's keep the HTTPS one. At some point the site will be updated.
```suggestion
5. [Separating Axis Theorem](https://programmerart.weebly.com/separating-axis-theorem.html)
```

File: files/en-us/games/tutorials/2d_breakout_game_phaser/randomizing_gameplay/index.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ You can do a lot more in the game — add whatever you feel would be best to mak
 - Grant extra bonus points if bricks are destroyed rapidly, several-in-a-row (or other bonuses of your choosing.)
 - Create levels with different brick layouts.
 
-Be sure to check the ever growing list of [examples](https://examples.phaser.io/) and the [official documentation](https://phaser.io/docs/), and visit the [HTML5 Gamedevs forums](https://www.html5gamedevs.com/forum/14-phaser/) if you ever need any help.
+Be sure to check the ever growing list of [examples](http://phaser.io/examples) and the [official documentation](https://phaser.io/docs/), and visit the [HTML5 Gamedevs forums](https://www.html5gamedevs.com/forum/14-phaser/) if you ever need any help.
```
Comment (teoli2003) [inline files/en-us/games/tutorials/2d_breakout_game_phaser/randomizing_gameplay/index.md:54]:
```suggestion
Be sure to check the ever-growing list of [examples](https://phaser.io/examples) and the [official documentation](https://phaser.io/docs/), and visit the [HTML5 Gamedevs forums](https://www.html5gamedevs.com/forum/14-phaser/) if you ever need any help.
```

File: files/en-us/games/tutorials/html5_gamedev_phaser_device_orientation/index.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ By the end of the tutorial you will have a fully functional demo game: [Cyber Or
...
+[Phaser](https://phaser.io/) is a framework for building desktop and mobile HTML5 games. It's quite new, but growing rapidly thanks to the passionate community involved in the development process. You can check it out [on GitHub](https://github.com/photonstorm/phaser) where it's open sourced, read the [online documentation](https://phaser.io/docs/) and go through the big collection of [examples](http://phaser.io/examples). The Phaser framework provides you with a set of tools that will speed up development and help handle generic tasks needed to complete the game, so you can focus on the game idea itself.
```
Comment (teoli2003) [inline files/en-us/games/tutorials/html5_gamedev_phaser_device_orientation/index.md:24]:
```suggestion
[Phaser](https://phaser.io/) is a framework for building desktop and mobile HTML5 games. It's quite new, but growing rapidly thanks to the passionate community involved in the development process. You can check it out [on GitHub](https://github.com/photonstorm/phaser) where it's open sourced, read the [online documentation](https://phaser.io/docs/) and go through the big collection of [examples](https://phaser.io/examples). The Phaser framework provides you with a set of tools that will speed up development and help handle generic tasks needed to complete the game, so you can focus on the game idea itself.
```

File: files/en-us/mdn/guidelines/video/index.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ There are several arguments against using video content for technical documentat
 
 - Video is linear.
   People don't tend to read online documentation in a linear fashion, starting at the start and reading through to the end.
-  [They scan](https://www.sensible.com/chapter.html).
+  [They scan](http://sensible.com/dont-make-me-think/).
```
Comment (teoli2003) [inline files/en-us/mdn/guidelines/video/index.md:20]: The link seems to be a commercial one. Let's just remove it.
```suggestion
  _They scan._
```

File: files/en-us/mozilla/firefox/releases/1.5/what_s_new_in_1.5_alpha/index.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ tags:
 ---
 {{FirefoxSidebar}}
 
-This page is based largely on [http://www.squarefree.com/burningedg...eases/1.1.html](https://www.squarefree.com/burningedge/releases/1.1.html) (thanks Jesse).
+This page is based largely on [http://www.squarefree.com/burningedg...eases/1.1.html](http://www.squarefree.com/burningedge/releases/) (thanks Jesse).
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/1.5/what_s_new_in_1.5_alpha/index.md:16]:
```suggestion
This page is based largely on [https://www.squarefree.com/burningedg...eases/](https://www.squarefree.com/burningedge/releases/) (thanks Jesse).
```

File: files/en-us/web/accessibility/understanding_colors_and_luminance/index.md [hunk 6]
```diff
@@ -109,7 +109,7 @@ While text that is too small is hard to read, so is text that is too big. Above
 
 The current WCAG 2.x contrast guidelines 1.4.3 and 1.4.6 define "large" text as text that is 18pt (24px) or larger at a normal weight (400), or text that is 14pt (18.7px) at a bold weight (700). This may have been sufficient in 2008 when the standard was created. At that time, most "web fonts" such as Verdana were only available in a normal and bold weight.
 
-Today, the proliferation of easily available web fonts of all sizes and weights has complicated the issue. The emerging WCAG 3.0 standard on visual contrast is known as the [Advanced Perceptual Contrast Algorithm (APCA)](https://www.myndex.com/APCA/simple), and takes into account font weights and sizes in a variable manner, relative to the predicted contrast value.
+Today, the proliferation of easily available web fonts of all sizes and weights has complicated the issue. The emerging WCAG 3.0 standard on visual contrast is known as the [Advanced Perceptual Contrast Algorithm (APCA)](http://www.myndex.com/APCA/), and takes into account font weights and sizes in a variable manner, relative to the predicted contrast value.
```
Comment (teoli2003) [inline files/en-us/web/accessibility/understanding_colors_and_luminance/index.md:112]:
```suggestion
Today, the proliferation of easily available web fonts of all sizes and weights has complicated the issue. The emerging WCAG 3.0 standard on visual contrast is known as the [Advanced Perceptual Contrast Algorithm (APCA)](https://www.myndex.com/APCA/), and takes into account font weights and sizes in a variable manner, relative to the predicted contrast value.
```

File: files/en-us/web/guide/audio_and_video_delivery/cross-browser_audio_basics/index.md [hunk 7]
```diff
@@ -627,4 +627,4 @@ The following tables list basic audio support across desktop and mobile browsers
 - [Using HTML5 audio and video](/en-US/docs/Learn/HTML/Multimedia_and_embedding/Video_and_audio_content)
 - [Creating a cross-browser video player](/en-US/docs/Web/Guide/Audio_and_video_delivery/cross_browser_video_player)
 - [jPlayer](https://jPlayer.org): An open source audio and video library for jQuery and Zepto.
-- [Media Event Inspector](https://jplayer.org/HTML5.Media.Event.Inspector)
+- [Media Event Inspector](http://jplayer.org/HTML5.Media.Event.Inspector/)
```
Comment (teoli2003) [inline files/en-us/web/guide/audio_and_video_delivery/cross-browser_audio_basics/index.md:630]:
```suggestion
- [Media Event Inspector](https://jplayer.org/HTML5.Media.Event.Inspector/)
```

File: files/en-us/web/http/configuring_servers_for_ogg_media/index.md [hunk 8]
```diff
@@ -44,7 +44,7 @@ Your server must also return `206: Partial Content` for the request `Range: byte
 
 When the browser seeks through Ogg media to a specified time, it has to seek to the nearest key frame before the seek target, then download and decode the video from there until the requested target time. The farther apart your key frames are, the longer this takes, so it's helpful to include key frames at regular intervals.
 
-By default, [`ffmpeg2theora`](https://v2v.cc/~j/ffmpeg2theora/) uses one key frame every 64 frames (or about every 2 seconds at 30 frames per second), which works pretty well.
+By default, [`ffmpeg2theora`](http://v2v.cc/~j/ffmpeg2theora/) uses one key frame every 64 frames (or about every 2 seconds at 30 frames per second), which works pretty well.
```
Comment (teoli2003) [inline files/en-us/web/http/configuring_servers_for_ogg_media/index.md:47]:
```suggestion
By default, [`ffmpeg2theora`](https://v2v.cc/~j/ffmpeg2theora/) uses one key frame every 64 frames (or about every 2 seconds at 30 frames per second), which works pretty well.
```

File: files/en-us/web/http/link_prefetching_faq/index.md [hunk 9]
```diff
@@ -108,7 +108,7 @@ It is important that websites adopt `<link>` tag based prefetching instead of tr
 
 ### Which browsers support link prefetching?
 
-Browsers based on Mozilla 1.2 (or later) as well as browsers based on Mozilla 1.0.2 (or later) support prefetching. This includes Firefox and Netscape 7.01+. Camino builds as of March 2003 are based on Mozilla 1.0.1, and therefore do not support prefetching. [Test](https://gemal.dk/browserspy/prefetch.php) your browser to see if it supports Link Prefetching.
+Browsers based on Mozilla 1.2 (or later) as well as browsers based on Mozilla 1.0.2 (or later) support prefetching. This includes Firefox and Netscape 7.01+. Camino builds as of March 2003 are based on Mozilla 1.0.1, and therefore do not support prefetching. [Test](http://browserspy.dk/prefetch.php) your browser to see if it supports Link Prefetching.
```
Comment (teoli2003) [inline files/en-us/web/http/link_prefetching_faq/index.md:111]:
```suggestion
Browsers based on Mozilla 1.2 (or later), as well as browsers based on Mozilla 1.0.2 (or later), support prefetching. This includes Firefox and Netscape 7.01+. Camino builds as of March 2003 are based on Mozilla 1.0.1, and therefore do not support prefetching. [Test](https://browserspy.dk/prefetch.php) your browser to see if it supports Link Prefetching.
```

File: files/en-us/web/svg/index.md [hunk 10]
```diff
@@ -52,5 +52,5 @@ See also, [SVG Tutorial](/en-US/docs/Web/SVG/Tutorial)
 - [SVG authoring guidelines](https://jwatt.org/svg/authoring/)
 - [SVG as an image](/en-US/docs/Web/SVG/SVG_as_an_Image)
 - [SVG animation with SMIL](/en-US/docs/Web/SVG/SVG_animation_with_SMIL)
-- [SVG art gallery](https://plurib.us/1shot/2007/svg_gallery/)
+- [SVG art gallery](http://www1.plurib.us/svg_gallery/)
```
Comment (teoli2003) [inline files/en-us/web/svg/index.md:55]:
```suggestion
- [SVG art gallery](https://www1.plurib.us/svg_gallery/)
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
So this one is trickier:
- We prefer not to have redirects
- But we don't like http links. 
So we want to have https links that doesn't redirect.
In case there aren't any, we prefer having an https link that redirect to an http link because, when they will fix their website to support https correctly, the link will be automatically fixed while the http one will not.

**Comment by OnkarRuikar** [comment]
>   In case there aren't any, we prefer having an https link that redirect to an http link because, when they will fix their website to support https correctly, the link will be automatically fixed while the http one will not.

I see. Mostly they go from `https` to `http` because they don't want to pay for the certificate anymore. This way when they renew the certificate the URL will stop redirecting.

Counter argument can be made that, taking users from `https` to `http` on same domain is considered deceptive. It'll look bad for those domains.

**Comment by teoli2003** [comment]
> I see. Mostly they go from `https` to `http` because they don't want to pay for the certificate anymore. This way when they renew the certificate the URL will stop redirecting.

There are free certificates nowadays. (See https://letsencrypt.org/ )

**Review by teoli2003 — APPROVED** [review]
Approving, but waiting for Eric's review before merging.

**Comment by teoli2003** [comment]
I'm changing strategy here: I will merge this to prevent the rotting of the bug. If Eric finds any problem, we'll fix it in a follow-up.

**Pull request merged**